src/account/account.cairo

```diff
@@ -160,6 +153,23 @@ mod Account {
         }
     }
 
+    #[external(v0)]
+    impl SRC6CamelOnlyImpl of interface::ISRC6CamelOnly<ContractState> {
+        fn isValidSignature(
+            self: @ContractState, hash: felt252, signature: Array<felt252>
+        ) -> felt252 {
+            SRC6Impl::is_valid_signature(self, hash, signature)
+        }
+    }
+
+    #[external(v0)]
+    impl SRC5CamelImpl of ISRC5Camel<ContractState> {
```

Comment:
hm why one is `SRC6CamelOnlyImpl` and the other `SRC5CamelImpl`? we should normalize these names

Reply:
We are using CamelOnly when methods some are not added (having single-word function names), and just Camel, when all the methods are added (with the different casing).